backend/builder.go

```diff
@@ -152,7 +152,6 @@ func Wire(d *dagger.Client, src *dagger.Directory, platform dagger.Platform, goV
 		WithExec([]string{"apk", "add", "make"}).
 		WithMountedDirectory("/src", src).
 		WithWorkdir("/src").
-		WithEnvVariable("WIRE_TAGS", wireTag).
-		WithExec([]string{"make", "gen-go"}).
+		WithExec([]string{"/bin/sh", "-c", fmt.Sprintf("make gen-go WIRE_TAGS=%s", wireTag)}).
```

Comment:
why does this need `sh -c`? wouldn't this be enough?

```suggestion
		WithExec([]string{"make", "gen-go", fmt.Sprintf("WIRE_TAGS=%s", wireTag)}).
```

Reply:
I honestly don't know but I can try it and see if it does.

Reply:
yeah that works tooa nd I prefer not invoking it from `/bin/sh` anyways. 

```
/go/bin/wire-v0.5.0 gen -tags pro ./pkg/server
```